blocks/api/raw-handling/index.js

```diff
@@ -150,33 +187,21 @@ export default function rawHandler( { HTML, plainText = '', mode = 'AUTO', tagNa
 
 		doc.body.innerHTML = piece;
 
-		const transformsFrom = getBlockTransforms( 'from' );
-
-		const blocks = Array.from( doc.body.children ).map( ( node ) => {
-			const transformation = findTransform( transformsFrom, ( transform ) => (
-				transform.type === 'raw' &&
-				transform.isMatch( node )
-			) );
-
-			if ( transformation ) {
-				if ( transformation.transform ) {
-					return transformation.transform( node );
-				}
-
-				return createBlock(
-					transformation.blockName,
-					getBlockAttributes(
-						getBlockType( transformation.blockName ),
-						node.outerHTML
-					)
-				);
+		return Array.from( doc.body.children ).map( ( node ) => {
+			const { transform, blockName } =
+				find( rawTransformations, ( { isMatch } ) => isMatch( node ) );
```

Comment:
Adjusted so that if someone forgets to add a selector:

> A block registered a raw transformation schema for `H2` but did not match it. Make sure there is a `selector` or `isMatch` property that can match the schema.
> Sanitized HTML: `<h2>Issue Overview</h2>`